app/controllers/labware_controller.rb

```diff
@@ -35,7 +36,7 @@ def update # rubocop:todo Metrics/AbcSize
     notice = +"Labware: #{params[:labware_barcode]} has been changed to a state of #{params[:state].titleize}."
     notice << ' The customer will still be charged.' if update_params[2]
 
-    respond_to { |format| format.html { redirect_to(search_path, notice: notice) } }
+    respond_to { |format| format.html { redirect_to(search_path, notice:) } }
```

Comment:
I had forgotten about this syntax for Ruby.  There are probably lots of times I coud have shortened to this in my current work.